torch/csrc/distributed/autograd/rpc_messages/cleanup_autograd_context_req.cpp

```diff
@@ -14,10 +14,12 @@ int64_t CleanupAutogradContextReq::getContextId() {
 }
 
 rpc::Message CleanupAutogradContextReq::toMessage() && {
+  rpc::isInRpcCall = true;
   // pickle context_id using JIT pickler.
   std::vector<torch::Tensor> tensorTable;
   std::vector<char> payload =
       jit::pickle(at::IValue(context_id_), &tensorTable);
+  rpc::isInRpcCall = false;
```

Comment:
Is the set/unset needed here? Doesn't seem like we are pickling an RRef.

Reply:
yes, it ties to jit::pickle, I will rename it. 

when jit pickle the rref, it can check the isInRpcCall flag. I think we should reset it as well, otherwise if user call torch.save() afterward in the same thread, users can pickle rref outside  rpc